gcloud/orb.yaml

```diff
@@ -268,6 +269,30 @@ commands:
           unset TWINE_NON_INTERACTIVE
           twine upload --repository-url <<parameters.repository-url>> <<parameters.path>>/<<parameters.files>>
 
+  overwrite-gcs-folder:
```

Comment:
Some general commentary.. This seems like a very niche use case. How often does one need to nuke a folder within GCS? Does a generalized command for this belong in the gcloud orb?

Reply:
Well, we do for every docs build! I think this is a slightly more generalized form than having the `docs/compile-and-upload` step make the assumption that your docs live in GCS as well as being a bit more configureable for us.

It is pretty niche, but I don't really see an issue with that in this case. Still beats copying and pasting these lines into all of our repos :P

Reply:
Yeah I was thinking it sounds like something more belonging in docs, but it is also true our docs orb is tailored to our use case 🤷

Reply:
Ah, for a bit more potential context on that, we're having to update the way we do documention anyway to be a bit more repo-local (see my draft [here](https://github.com/talkiq/realtime-ctm/pull/593) where I'm working towards that), which kind of pushed me away from the `docs` orb making much sense.

Reply:
Ah thanks!